osbs/core.py

```diff
@@ -841,6 +842,16 @@ def get_image_stream_tag(self, tag_id):
         check_response(response, log_level=logging.DEBUG)
         return response
 
+    @retry_on_not_found
+    def get_image_stream_tag_with_retry(self, tag_id):
```

Comment:
We don't have any test coverage for this -- is this ok?

Reply:
that should be covered by https://github.com/containerbuildsystem/osbs-client/pull/911/files#diff-625efbbf35a2290861f596fee1b19838R2172

not sure why it is showing that it isn't

Reply:
nvm that test is testing method in api, and we are always mocking methods in core